<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Andreas Bremges</title>

    <meta name="description" content="Postdoctoral researcher in Alice McHardy's group, Helmholtz Centre for Infection Research. Previously Ph.D. student in Alex Sczyrba's group, Bielefeld University.">
    <meta name="keywords" content="bioinformatics, metagenomics, single cell genomics, microbial dark matter">

    <link rel="stylesheet" href="tufte.css"/>
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
      ga('create', 'UA-60416689-1', 'auto');
      ga('send', 'pageview');
    </script>
  </head>

  <body>
    <article>
      <h1 id="abremges">Andreas Bremges</h1>
      <p class="subtitle">Postdoctoral researcher</p>
      <section>
        <p>I'm a bioinformatician&mdash;I develop efficient methods and algorithms for biological sequence analysis and apply these techniques to study microbial communities from complex environments.</p>
      </section>

      <section>
        <h2 id="publications"><br>Selected publications (more on <a href="https://scholar.google.de/citations?user=iASXbxgAAAAJ&hl=en" target="_blank">Google Scholar</a>)</h2>
        <div class="epigraph">
          <blockquote>
            <p>Genomics and prevalence of bacterial and archaeal isolates from biogas-producing microbiomes</p>
            <footer>Maus, I.*, <b>A. Bremges*</b>, &hellip;, A. Pühler, &amp; A. Schlüter.<br><cite>Biotechnology for Biofuels</cite>, 2017. doi:<a href="https://doi.org/10.1186/s13068-017-0947-1" target="_blank">10.1186/s13068-017-0947-1</a></footer>
          </blockquote>
          <blockquote>
            <p>Critical Assessment of Metagenome Interpretation&mdash;a benchmark of metagenomics software</p>
            <footer>Sczyrba, A.*, P. Hofmann*, P. Belmann*, &hellip;, <b>A. Bremges</b>, &hellip;, T. Rattei, &amp; A.C. McHardy.<br><cite>Nature Methods</cite>, 2017. doi:<a href="https://doi.org/10.1038/nmeth.4458" target="_blank">10.1038/nmeth.4458</a></footer>
          </blockquote>
          <blockquote>
            <p>Metagenomics and CAZyme discovery</p>
            <footer>Kunath, B.J., <b>A. Bremges</b>, A. Weimann, A.C. McHardy, &amp; P.B. Pope.<br><cite>Methods in Molecular Biology</cite>, 2017. doi:<a href="https://doi.org/10.1007/978-1-4939-6899-2_20" target="_blank">10.1007/978-1-4939-6899-2_20</a></footer>
          </blockquote>
          <blockquote>
            <p>From genomes to phenotypes: Traitar, the microbial trait analyzer</p>
            <footer>Weimann, A., K. Mooren, J. Frank, P.B. Pope, <b>A. Bremges</b>, &amp; A.C. McHardy.<br><cite>mSystems</cite>, 2016. doi:<a href="https://doi.org/10.1128/mSystems.00101-16" target="_blank">10.1128/mSystems.00101-16</a></footer>
          </blockquote>
          <blockquote>
            <p>MeCorS: Metagenome-enabled error correction of single cell sequencing reads</p>
            <footer><b>Bremges, A.</b>, E. Singer, T. Woyke, &amp; A. Sczyrba.<br><cite>Bioinformatics</cite>, 2016. doi:<a href="https://doi.org/10.1093/bioinformatics/btw144" target="_blank">10.1093/bioinformatics/btw144</a></footer>
          </blockquote>
        </div>
      </section>

      <section>
        <h2 id="contact">Contact</h2>
        <p><em>Twitter:</em> <a href="https://twitter.com/abremges" target="_blank">@abremges</a></p>
        <p><em>Email:</em> <a href="mailto:andreas@bremges.de">andreas@bremges.de</a></p>
      </section>

    </article>
  </body>
</html>
