---
title: "Publications"
layout: page
---
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

<div class="pub">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1038/s41598-017-17312-2'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1038/s41598-017-17312-2" target="_blank">
Investigation of different nitrogen reduction routes and their key microbial players in wood chip-driven denitrification beds.
</a></h3>
Grie√ümeier, Bremges, <i>et al.</i> <b>(2017)</b>. <i>Scientific Reports</i>, 7, 1:17028.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1111/1751-7915.12982'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1111/1751-7915.12982" target="_blank">
Targeted <i>in situ</i> metatranscriptomics for selected taxa from mesophilic and thermophilic biogas plants.
</a></h3>
Stolze, Bremges, <i>et al.</i> <b>(2017)</b>. <i>Microbial Biotechnology</i>, in press.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/s13068-017-0947-1'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/s13068-017-0947-1" target="_blank">
Genomics and prevalence of bacterial and archaeal isolates from biogas-producing microbiomes.
</a></h3>
Maus*, Bremges*, <i>et al.</i> <b>(2017)</b>. <i>Biotechnology for Biofuels</i>, 10:264.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1038/nmeth.4458'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1038/nmeth.4458" target="_blank">
Critical Assessment of Metagenome Interpretation&mdash;a benchmark of metagenomics software.
</a></h3>
Sczyrba*, Hofmann*, Belmann*, <i>et al.</i> <b>(2017)</b>. <i>Nature Methods</i>, 14, 11:1063&ndash;1071.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1101/166835'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1101/166835" target="_blank">
Fast and memory-efficient noisy read overlapping with KD-trees.
</a></h3>
Parkhomchuk*, Bremges*, <i>et al.</i> <b>(2017)</b>. <i>bioRxiv</i>, preprint.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1038/s41598-017-03315-6'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1038/s41598-017-03315-6" target="_blank">
Characterisation of a stable laboratory co-culture of acidophilic nanoorganisms.
</a></h3>
Krause, Bremges, <i>et al.</i> <b>(2017)</b>. <i>Scientific Reports</i>, 7, 1:3289.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1007/978-1-4939-6899-2_20'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1007/978-1-4939-6899-2_20" target="_blank">
Metagenomics and CAZyme discovery.
</a></h3>
Kunath, Bremges, <i>et al.</i> <b>(2017)</b>. <i>Methods in Molecular Biology</i>, 1588:255&ndash;277.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1128/mSystems.00101-16'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1128/mSystems.00101-16" target="_blank">
From genomes to phenotypes: Traitar, the microbial trait analyzer.
</a></h3>
Weimann <i>et al.</i> <b>(2016)</b>. <i>mSystems</i>, 1, 6:e00101&ndash;16
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1016/j.jbiotec.2016.05.001'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1016/j.jbiotec.2016.05.001" target="_blank">
Genomic characterization of <i>Defluviitoga tunisiensis</i> L3, a key hydrolytic bacterium in a thermophilic biogas plant and its abundance as determined by metagenome fragment recruitment.
</a></h3>
Maus <i>et al.</i> <b>(2016)</b>. <i>Journal of Biotechnology</i>, 232:50&ndash;60.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1016/j.jbiotec.2016.04.008'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1016/j.jbiotec.2016.04.008" target="_blank">
Finished genome sequence and methylome of the cyanide-degrading <i>Pseudomonas pseudoalcaligenes</i> strain CECT5344 as resolved by single-molecule real-time sequencing.
</a></h3>
Wiberg*, Bremges*, <i>et al.</i> <b>(2016)</b>. <i>Journal of Biotechnology</i>, 232:61&ndash;8.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/s13068-016-0581-3'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/s13068-016-0581-3" target="_blank">
Unraveling the microbiome of a thermophilic biogas plant by metagenome and metatranscriptome analysis complemented by characterization of bacterial and archaeal isolates.
</a></h3>
Maus <i>et al.</i> <b>(2016)</b>. <i>Biotechnology for Biofuels</i>, 9:171.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1016/j.jbiotec.2016.06.014'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1016/j.jbiotec.2016.06.014" target="_blank">
An integrated metagenome and -proteome analysis of the microbial community residing in a biogas production plant.
</a></h3>
Ortseifen <i>et al.</i> <b>(2016)</b>. <i>Journal of Biotechnology</i>, 231:268&ndash;279.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/s13068-016-0565-3'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/s13068-016-0565-3" target="_blank">
Identification and genome reconstruction of abundant distinct taxa in microbiomes from one thermophilic and three mesophilic production-scale biogas plants.
</a></h3>
Stolze*, Bremges*, <i>et al.</i> <b>(2016)</b>. <i>Biotechnology for Biofuels</i>, 9:156.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1093/bioinformatics/btw144'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1093/bioinformatics/btw144" target="_blank">
MeCorS: Metagenome-enabled error correction of single cell sequencing reads.
</a></h3>
Bremges <i>et al.</i> <b>(2016)</b>. <i>Bioinformatics</i>, 32, 14:2199&ndash;201.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/s13742-015-0087-0'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/s13742-015-0087-0" target="_blank">
Bioboxes: standardised containers for interchangeable bioinformatics software.
</a></h3>
Belmann <i>et al.</i> <b>(2015)</b>. <i>GigaScience</i>, 4:47.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1002/pmic.201400557'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1002/pmic.201400557" target="_blank">
Fractionation of biogas plant sludge material improves metaproteomic characterization to investigate metabolic activity of microbial communities.
</a></h3>
Kohrs <i>et al.</i> <b>(2015)</b>. <i>Proteomics</i>, 15, 20:3585&ndash;9.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/s13742-015-0073-6'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/s13742-015-0073-6" target="_blank">
Deeply sequenced metagenome and metatranscriptome of a biogas-producing microbial community from an agricultural production-scale biogas plant.
</a></h3>
Bremges <i>et al.</i> <b>(2015)</b>. <i>GigaScience</i>, 4:33.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1016/j.jbiotec.2014.02.004'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1016/j.jbiotec.2014.02.004" target="_blank">
Complete genome sequence of the cyanide-degrading bacterium <i>Pseudomonas pseudoalcaligenes</i> CECT5344.
</a></h3>
Wibberg <i>et al.</i> <b>(2014)</b>. <i>Journal of Biotechnology</i>, 175:67&ndash;8.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1371/journal.pone.0081912'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1371/journal.pone.0081912" target="_blank">
A silent exonic SNP in <i>Kdm3a</i> affects nucleic acids structure but does not regulate experimental autoimmune encephalomyelitis.
</a></h3>
Gillett <i>et al.</i> <b>(2013)</b>. <i>PLOS ONE</i>, 8, 12:e81912.
</div>

<div class="pub" style="margin-top:2em">
<div class='altmetric-embed' data-badge-type='donut' data-condensed="true" style="display:inline; float:right; margin-left:1em" data-doi='10.1186/1471-2105-11-222'></div>
<h3 style="margin:0">
<a href="https://doi.org/10.1186/1471-2105-11-222" target="_blank">
Fine-tuning structural RNA alignments in the twilight zone.
</a></h3>
Bremges*, Schirmer*, <i>et al.</i> <b>(2010)</b>. <i>BMC Bioinformatics</i>, 11:222.
</div>
