<br/><br/>
**<a href="https://github.com/metagenomics/kgrep" target="_blank">Metagenomic proxy assemblies of single cell genomes</a>**  
__Bremges, A__, Jarett, J, Woyke, T, Sczyrba, A **(2016)**. *In preparation.*

<br/><br/>
**<a href="https://github.com/abremges/2015-pseudo" target="_blank">Finished genome sequence and methylome of the cyanide-degrading *Pseudomonas pseudoalcaligenes* strain CECT5344 as resolved by single-molecule real-time sequencing</a>**  
Wibberg, D\*, __Bremges, A\*__, Dammann-Kalinowski, T, Maus, I, Igeño, I, Vogelsang, R, König, C, Sczyrba, A, Blasco, R, Pühler, A, Schlüter, A **(2016)**. *Submitted.*

<br/><br/>
**<a href="https://www.youtube.com/watch?v=DLzxrzFCyOs" target="_blank">Genomic characterization of *Defluviitoga tunisiensis* L3, a key hydrolytic bacterium in a thermophilic biogas plant and its abundance as determined by metagenome fragment recruitment</a>**  
Maus, I, Cibis KG, __Bremges, A__, Stolze, Y, Wibberg, D, Tomazetto, G, Blom, J, Sczyrba, A, König, H, Pühler, A, Schlüter, A **(2016)**. *Submitted.*

<br/><br/>
**<a href="https://www.youtube.com/watch?v=DLzxrzFCyOs" target="_blank">Integrative metagenome/metaproteome analysis for a biogas production plant revealed abundantly expressed proteins and their genomic context</a>**  
Ortseifen, V, Stolze, Y, Maus, I, Sczyrba, A, __Bremges, A__, Albaum, S, Jaenicke, S, Fracowiak, J, Pühler, A, Schlüter, A **(2016)**. *In preparation.*

<br/><br/>
**<a href="https://www.youtube.com/watch?v=DLzxrzFCyOs" target="_blank">Identification and genome reconstruction of distinct taxa in microbiomes  from four different production-scale biogas plants</a>**  
Stolze, Y\*, __Bremges, A\*__, Maus, I, Rumming, M, Pühler, A, Sczyrba, A, Schlüter, A **(2016)**. *In preparation.*

<br/><br/>
**<a href="https://www.youtube.com/watch?v=DLzxrzFCyOs" target="_blank">Towards the core microbiome for a thermophilic biogas plant applying metagenome and metatranscriptome analyses complemented by cultivation and characterization of isolates for major functional groups</a>**  
Maus, I, Koeck, DE, Cibis, K, Hahnke, S, Kim, YS, Langer, T, Kreubel, J, Erhard, M, __Bremges, A__, Off, S, Stolze, Y, Jaenicke, S, Sczyrba, A, Scherer, P, König, H, Schwarz, WH, Pühler, A, Schlüter, A, Klocke, M **(2016)**. *In preparation.*
