---
title: "publications"
bg: gray2
color: black
fa-icon: navicon
---

**<a href="http://dx.doi.org/10.1101/043315" target="_blank">From genomes to phenotypes: Traitar, the microbial trait analyzer</a>**  
Weimann, A., J. Frank, P.B. Pope, __A. Bremges__, A.C. McHardy **(2016)**. *bioRxiv*, preprint.

<br/><br/>
**<a href="http://dx.doi.org/10.1016/j.jbiotec.2016.05.001" target="_blank">Genomic characterization of *Defluviitoga tunisiensis* L3, a key hydrolytic bacterium in a thermophilic biogas plant and its abundance as determined by metagenome fragment recruitment</a>**  
Maus, I., K.G. Cibis, __A. Bremges__, Y. Stolze, D. Wibberg, G. Tomazetto, J. Blom, A. Sczyrba, H. König, A. Pühler, A. Schlüter **(2016)**. *Journal of Biotechnology*, in press.

<br/><br/>
**<a href="http://dx.doi.org/10.1016/j.jbiotec.2016.04.008" target="_blank">Finished genome sequence and methylome of the cyanide-degrading *Pseudomonas pseudoalcaligenes* strain CECT5344 as resolved by single-molecule real-time sequencing</a>**  
Wibberg, D.\*, __A. Bremges\*__, T. Dammann-Kalinowski, I. Maus, I. Igeño, R. Vogelsang, C. König, A. Sczyrba, R. Blasco, A. Pühler, A. Schlüter **(2016)**. *Journal of Biotechnology*, in press.

<br/><br/>
**<a href="http://dx.doi.org/10.1093/bioinformatics/btw144" target="_blank">MeCorS: metagenome-enabled error correction of single cell sequencing reads</a>**  
__Bremges, A.__, E. Singer, T. Woyke, A. Sczyrba **(2016)**. *Bioinformatics*, in press.

<br/><br/>
**<a href="http://dx.doi.org/10.1186/s13742-015-0087-0" target="_blank">Bioboxes: standardised containers for interchangeable bioinformatics software</a>**  
Belmann, P., J. Dröge, __A. Bremges__, A.C. McHardy, A. Sczyrba, M.D. Barton **(2015)**. *GigaScience*, 4:47.

<br/><br/>
**<a href="http://dx.doi.org/10.1186/s13742-015-0073-6" target="_blank">Deeply sequenced metagenome and metatranscriptome of a biogas-producing microbial community from an agricultural production-scale biogas plant</a>**  
__Bremges, A.__, I. Maus, P. Belmann, F.G. Eikmeyer, A. Winkler, A. Albersmeier, A. Pühler, A. Schlüter\*, A. Sczyrba\* **(2015)**. *GigaScience*, 4:33.

<br/><br/>
**<a href="http://dx.doi.org/10.1002/pmic.201400557" target="_blank">Fractionation of biogas plant sludge material improves metaproteomic characterization to investigate metabolic activity of microbial communities</a>**  
Kohrs, F., S. Wolter, D. Benndorf, R. Heyer, M. Hoffmann, E. Rapp, __A. Bremges__, A. Sczyrba, A. Schlüter, U. Reichl **(2015)**. *Proteomics*, 15, 20:3585-9.

<br/><br/>
**<a href="http://dx.doi.org/10.1016/j.jbiotec.2014.02.004" target="_blank">Complete genome sequence of the cyanide-degrading bacterium *Pseudomonas pseudoalcaligenes* CECT5344</a>**  
Wibberg, D., V.M. Luque-Almagro, I. Igeño, __A. Bremges__, M.D. Roldán, F. Merchán, L.P. Sáez, M.I. Guijo, M.I. Manso, D. Macías, P. Cabello, G. Becerra, M.I. Ibáñez, M.I. Carmona, M.M. Escribano, F. Castillo, A. Sczyrba, C. Moreno-Vivián, R. Blasco, A. Pühler, A. Schlüter **(2014)**. *Journal of Biotechnology*, 175:67-8.

<br/><br/>
**<a href="http://dx.doi.org/10.1371/journal.pone.0081912" target="_blank">A silent exonic SNP in *kdm3a* affects nucleic acids structure but does not regulate experimental autoimmune encephalomyelitis</a>**  
Gillett, A., P. Bergman, R. Parsa, __A. Bremges__, R. Giegerich, M. Jagodic **(2013)**. *PLoS ONE*, 8, 12:e81912.

<br/><br/>
**<a href="http://dx.doi.org/10.1186/1471-2105-11-222" target="_blank">Fine-tunig structural RNA alignments in the twilight zone</a>**  
__Bremges, A.\*__, S. Schirmer\*, R. Giegerich **(2010)**. *BMC Bioinformatics*, 11:222.
